CONTRIBUTING.md

```diff
@@ -62,9 +69,18 @@ This repo is a “monorepo,” meaning it contains several projects in one. It c
    pnpm i
    ```
 
-#### Developing using Gitpod
+#### Developing using GitHub Codespaces
+
+To get started, create a codespace for this repository by clicking this 👇
 
-Instead of working locally on your machine, you can also contribute using an online coding development environment like Gitpod.
```

Comment:
I think this line was important to distinguish the Gitpod instructions from the local ones.

How about a “Developing online” heading followed by a variation of this sentence before we get to the GitHub and Gitpod headings? (Those would then become h5 instead of h4.)